controllers/ibu_controller.go

```diff
@@ -248,7 +248,7 @@ func (r *ImageBasedUpgradeReconciler) handleAbortOrFinalize(ctx context.Context,
 		case string(utils.ConditionReasons.FinalizeFailed):
 			nextReconcile, err = r.handleFinalizeFailure(ctx, ibu)
 		}
-		if nextReconcile.Requeue == false {
+		if nextReconcile.Requeue == false && idleCondition.Reason == string(utils.ConditionReasons.Idle) {
```

Comment:
With the new condition, the old one can be removed? Then we don't have to requeue for failure handling? Annotation would be an edit that will trigger reconcile. Originally we were thinking about some sort of flag file which would require polling.

Reply:
I am not sure what you mean by new and old conditions?

Adding annotation does not trigger reconcile on ibu.

Reply:
removed the condition and add trigger to reconcile upon adding or removing this annotation